meta-ostro/conf/local.conf.sample

```diff
@@ -291,6 +291,18 @@ SUPPORTED_RECIPES_CHECK = "fatal"
 #
 #require conf/distro/include/ostro-os-production.inc
 
+# systemd-bootchart is a useful tool to analyze and optimize a system
+# boot time. The tool is available in Ostro and needs to be activated
+# by a kernel command-line parameter which requires to build a new
+# image. You also need to make sure that you enable CONFIG_SCHEDSTATS=y
+# (that will also set CONFIG_SCHED_INFO=y) in your kernel, this option
+# is not enabled by default. For instructions on how to modify an Ostro
+# kernel, please check doc/howtos/modifying-ostro-kernel.rst.
```

Comment:
OK, it looks like I just wasn't patient enough... ;-/
It does get created but it takes a little while to complete (20 sec with the default settings).

I have verified this on both intel-quark and intel-corei7-64 so far, I will update this PR to includes the lines (commented out by default) that include the fragment file. The PR with the fragments will be sent in a separate PR.